CMakeLists.txt

```diff
@@ -317,6 +277,19 @@ endif()
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
 set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD ${CMAKE_CXX_STANDARD})
 
+# Skip signing the code - otherwise iOS will not compile in CI at all! Newer versions
```

Comment:
trivial: Reword this? There's some repetition in the description. I've seen both conversational and matter of fact comments throughout our code and don't feel strongly either way but it may be clearer and to the point to remove excess wording? e.g. "This only really applies to iOS currently, as MacOS isn't as picky about it" -> "This only applies to iOS as MacOS doesn't care.". Very nitpicking here -_-;;